patches/chrome-installer-util-google_update_settings.cc.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/chrome/installer/util/google_update_settings.cc b/chrome/installer/util/google_update_settings.cc
+index 42a2c0cc1b9576aa5f6c03e615e4b8450611831d..c2c8cc5e7d0bbdc6bf1b073ccfeec4c547497fff 100644
+--- a/chrome/installer/util/google_update_settings.cc
++++ b/chrome/installer/util/google_update_settings.cc
+@@ -36,7 +36,11 @@ using base::win::RegKey;
+ using installer::InstallationState;
+ 
+ const wchar_t GoogleUpdateSettings::kPoliciesKey[] =
++#if defined(BRAVE_CHROMIUM_BUILD) && defined(OFFICIAL_BUILD)
++    L"SOFTWARE\\Policies\\BraveSoftware\\Update";
++#else
+     L"SOFTWARE\\Policies\\Google\\Update";
++#endif
```

Comment:
Continue to patch for this part, later I think we will add some new way to do string replacements for overrides.  I imagine it something like we copy from chromium_src to some gen dir, and we include from there after applying some overrides.   Different task though.